app/services/foreman/plugin.rb

```diff
@@ -243,6 +243,27 @@ def role(name, permissions)
       end
     end
 
+    # Add plugin permissions to core's Manager and Viewer roles
+    # Usage:
+    # add_permissions_to_core_roles ['MyPlugin::FirstResource', 'MyPlugin::SecondResource'], :except => [:skip_this_permission]
+    def add_resource_permissions_to_core(resources, opts = {})
```

Comment:
If the roles are referred to as the "default roles" everywhere else, can we name this method similarly? e.g. add_resource_permissions_to_default_roles 

I think that would make things more readable/obvious.